
int32 object_id
string object_name
geometry_msgs/Point position